pyproject.toml

```
@@ -27,16 +27,16 @@ url = "https://epc-power-us-890711758583.d.codeartifact.us-east-2.amazonaws.com/
 secondary = true
 
 [tool.poetry.dependencies]
-python = ">=3.7.9,<3.11"
+python = ">=3.8,<3.11"
 
 #### EPC Repos ####
 #sunspecdemo = {version = "0.3.0", source = "epc-power"}
-sunspecdemo = { git = "https://github.com/epcpower/sunspec-demo.git", rev = "a04b466da431549946297c66364ee835c3fab3e0" }
+sunspecdemo = { git = "https://github.com/epcpower/sunspec-demo.git", rev = "57b7699ffc000ad5d2018748c7fdbe3273b7e908" }
 
 ###################
 
 attrs = "20.2.0"
-lxml = "4.3.0"
+lxml = "5.2.0"
```

Recent update. Might as well update it now.
```suggestion
lxml = "5.2.1"
```

---

[cf2cc72](https://github.com/epcpower/pm/pull/92/commits/cf2cc72b24f8178d440c354ce12855ca96ca955e)